src/Microsoft.SqlTools.ServiceLayer/ShowPlan/Contracts/ExecutionPlanGraph.cs

```diff
@@ -23,7 +23,11 @@ public class ExecutionPlanGraph
         /// <summary>
         /// Underlying xml string used for generating execution plan graph
         /// </summary>
-        public string XmlString { get; set; }
+        public string RawGraph { get; set; }
+        /// <summary>
+        /// Index recommendations given by show plan to improve query performance
+        /// </summary>
+        public List<ExecutionPlanRecommendation> Recommendations;
```

Comment:
Is there a reason why we aren't using `{get; set;}` for `Recommendations`? All of the other list properties in the other classes defined in this file have them.